src/OSDP.Net/ControlPanel.cs

```diff
@@ -24,7 +24,7 @@ namespace OSDP.Net
     /// </summary>
     public class ControlPanel
     {
-        private const byte ConfigurationAddress = 0x7f;
+        internal const byte ConfigurationAddress = 0x7f;
```

Comment:
just a comment

seems osdp protocol has a number of constants which are part of spec itself and they are shared by various pieces (i.e. PD and ACU).  Seems we could find a better, global and shared home for these constants